src/pkg/packager/filters/deploy_test.go

```diff
@@ -0,0 +1,128 @@
+package filters
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/defenseunicorns/zarf/src/pkg/utils/helpers"
+	"github.com/defenseunicorns/zarf/src/types"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func cfq(t *testing.T, q string) types.ZarfComponent {
```

Comment:
Component From Query, since this function is only used in testing, I felt an opaque name was fine. Can def make it more descriptive if that's your jam.